clang/docs/ReleaseNotes.rst

```diff
@@ -644,8 +646,8 @@ New features
   if class of allocation and deallocation function mismatches.
   `Documentation <https://clang.llvm.org/docs/analyzer/checkers.html#unix-mismatcheddeallocator-c-c>`__.
 
-- Function effects, e.g. the ``nonblocking`` and ``nonallocating`` "performance constraint" 
-  attributes, are now verified. For example, for functions declared with the ``nonblocking`` 
+- Function effects, e.g. the ``nonblocking`` and ``nonallocating`` "performance constraint"
```

Comment:
nit: Unrelated change.

Reply:
Ah, that’s my IDE automatically deleting a trailing space at the end of the line on save. I’ll fix that separately real quick if someone hasn’t already done so.

Reply:
Alright, fixed.